operate/client/src/App/Processes/MigrationView/MigrationSummaryNotification/index.tsx

```diff
@@ -7,44 +7,16 @@
  */
 
 import {MigrationSummary, InlineNotification} from './styled';
-import {processesStore} from 'modules/stores/processes/processes.migration';
-import {observer} from 'mobx-react';
-import {processInstanceMigrationStore} from 'modules/stores/processInstanceMigration';
-import pluralSuffix from 'modules/utils/pluralSuffix';
-
-const MigrationSummaryNotification: React.FC = observer(() => {
-  const {
-    migrationState: {selectedTargetProcess, selectedTargetVersion},
-    getSelectedProcessDetails,
-  } = processesStore;
-
-  const {
-    processName: selectedSourceProcessName,
-    version: selectedSourceProcessVersion,
-  } = getSelectedProcessDetails();
+import {MigrationDetails} from '../MigrationDetails';
 
+const MigrationSummaryNotification: React.FC = () => {
   return (
     <InlineNotification
       kind="info"
       title=""
       children={
         <MigrationSummary orientation="vertical" gap={5}>
-          <p>
-            You are about to migrate{' '}
-            {pluralSuffix(
-              processInstanceMigrationStore.state.selectedInstancesCount,
-              'process instance',
-            )}{' '}
-            from the process definition:{' '}
-            <strong>
-              {`${selectedSourceProcessName} - version ${selectedSourceProcessVersion}`}
-            </strong>{' '}
-            to the process definition:{' '}
-            <strong>
-              {`${selectedTargetProcess?.name} - version ${selectedTargetVersion}`}
-            </strong>
-          </p>
-
```

Comment:
This moved to a separate component `MigrationDetails` which is reused also in the the comfirmation modal.